### ISSUE TITLE
Update spanish translation for wiki keyboard shortcuts 

### ISSUE THREAD

**Issue by nvdaes** [edrlab/thorium-reader#1227]
I will update the corresponding article placed on my website with GitHub pages for a more comfortable reading by screen reader users, since GitHub wikis cannot set lang attribute, as far as i know:

https://nvdaes.github.io/epub7

Here is the updated wiki md file.

**Comment by nvdaes** [comment]
[Keyboard-shortcuts-(Spanish).zip](https://github.com/edrlab/thorium-reader/files/5386837/Keyboard-shortcuts-.Spanish.zip)

**Comment by danielweck** [comment]
Hello, I updated the wiki page: https://github.com/edrlab/thorium-reader/wiki/Keyboard-shortcuts-(Spanish)
Thank you so much for keeping an eye on this. Question: how do you keep track of changes? I want to make sure translators do not spend ages / waste time figuring out the wiki page diff when wanting to update documents in their language.
Are you aware that GitHub wiki pages have a history? See for example:
https://github.com/edrlab/thorium-reader/wiki/Keyboard-shortcuts/_history
...and for example the recent addition of the CTRL-F search keyboard shortcut was a simple addition:
https://github.com/edrlab/thorium-reader/wiki/Keyboard-shortcuts/_compare/c336eb5345546506e62ad3f8673aea6c106b6746...9bb98d5bdaf6fb63f7f79e9df67334527ee639e4
(I fear that the diff is not accessible though)

**Comment by nvdaes** [comment]
Thank you! I know about the history on wikis, but I used git log -p since I am used to run git locally, so I saw that there is just an addition and I get an idea of the line number changed. I use Notepad plus plus and, if needed, control plus g to move the cursor to the exact line where I want to go. I updated the spanish website on wikipages and I will take care of es.json and will send a PR.

Enviado desde mi iPhone

**Comment by danielweck** [comment]
Thank you Noelia. As usual, the empty strings `""` in the locale JSON files make it easy to find missing translated strings.

**Comment by nvdaes** [comment]
Thanks. I've created PR #1232

I'll share it with spanish speakers on mailing lists for transparency,
giving a chance to provide feedback.

2020-10-16 2:52 GMT+02:00, Daniel Weck <notifications@github.com>:
